src/main/kotlin/com/swm_standard/phote/service/ExamService.kt

```diff
@@ -359,6 +360,19 @@ class ExamService(
         return examsAsCreator + examsAsExaminee
     }
 
+    fun readSharedExamInfo(examId: UUID, memberId: UUID): ReadSharedExamInfoResponse {
+        val sharedExam = sharedExamRepository.findById(examId).orElseThrow { NotFoundException(fieldName = "examId") }
+        return ReadSharedExamInfoResponse(
+            examId = examId,
+            title = sharedExam.title,
+            startTime = sharedExam.startTime,
+            endTime = sharedExam.endTime,
+            capacity = sharedExam.capacity,
+            workbookId = sharedExam.workbook.id,
+            isWriter = sharedExam.member.id == memberId,
```

Comment:
```suggestion
            role =
                if (exam.member.id == memberId) {
                    ParticipationType.CREATOR
                } else {
                    ParticipationType.EXAMINEE
                },
```

이거 `readAllSharedExams` 기능이랑 통일감있게 이렇게 수정하는 거 어떨까욘 `ExamDto` 에도 suggestion 넣어둘게